x/ccv/types/keys.go

```diff
@@ -8,6 +8,12 @@ const (
 	// module supports
 	Version = "1"
 
+	// ProviderPortID is the default port id the provider CCV module binds to
```

Comment:
It may be a good idea to change the hardcoded string to ```"providerPort"``` and ```"consumerPort"``` or something similar, to avoid having the same value as [ModuleName](https://github.com/cosmos/interchain-security/blob/marius/281-cp-portid/x/ccv/provider/types/keys.go#L18) for example

Reply:
I don't think it's necessary. AFAIK, there is no risk of collision between IBC ports and module names. See the ibc-go implementation of ICS-20 https://github.com/cosmos/ibc-go/blob/05c4bfbbebb20d1cc078ddf1d29dadc766d9251b/modules/apps/transfer/types/keys.go#L12